resources/startPage.html.template

```diff
@@ -83,9 +83,28 @@
             </div>
         </div>
         <hr />
+        {{#if showWhatsNew}}
         <div class="row">
             <div class="icon">
-                <i class="codicon codicon-new-file" aria-label="Add Docker files"></i>
+                <i class="codicon codicon-star-empty" alt=""></i>

```

Comment:
Since these images are purely decorative, I changed from `aria-label="..."` to `alt=""`, which will cause the images to be skipped by screen readers, which is desirable.